frontend/packages/integration-tests-cypress/tests/app/auth-multi-user-login.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../support';
+import { guidedTour } from '../../views/guided-tour';
+import { nav } from '../../views/nav';
+import { masthead } from '../../views/masthead';
+
+describe('Auth test', () => {
+  const KUBEADMIN_IDP = 'kube:admin';
+  const KUBEADMIN_USERNAME = 'kubeadmin';
+
+  afterEach(() => {
+    checkErrors();
+    cy.logout();
+  });
+
+  if (Cypress.env('BRIDGE_KUBEADMIN_PASSWORD')) {
+    it('logs in as test user via htpasswd identity provider, verifies sidenav menus, switches perspectives', () => {
+      const idp = Cypress.env('BRIDGE_HTPASSWD_IDP') || 'test';
+      const username = Cypress.env('BRIDGE_HTPASSWD_USERNAME') || 'test';
+      const passwd = Cypress.env('BRIDGE_HTPASSWD_PASSWORD') || 'test';
+      cy.login(idp, username, passwd);
+      cy.url().should('include', Cypress.env('appHost'));
+      guidedTour.close();
+      masthead.username.shouldHaveText(username);
+
+      cy.log('switches from dev to admin perspective');
+      nav.sidenav.switcher.shouldHaveText('Developer');
+      nav.sidenav.switcher.changePerspectiveTo('Administrator');
+      nav.sidenav.switcher.shouldHaveText('Administrator');
+
+      cy.log('does not show admin nav items in Administration to test user');
```

Comment:
Fixed. Adjusted so that when running Cypress we expose Redux store for debugging ie: `window.store = store`, along with [cypress-pipe](https://github.com/NicholasBoll/cypress-pipe) plug in, which enables us to retry until all feature flags used in the side nav menus are defined, before attempting to test the side nav menu contents.